configs/cluster/icds-14lakh_reduced.yml

```diff
@@ -478,7 +478,7 @@ services:
   pg_warehouse_citus:
     # this should have replica nodes
     usage_capacity_per_node: 150000
-    include_ha_resources: True
+    # include_ha_resources: True
```

Comment:
Please don't change this file, it is used for ICDS sizing. Feel free to make a copy and update the copy.

Reply:
Reverted here: https://github.com/dimagi/commcare_resource_model/pull/25/commits/eb22edca1ad84d384052d982107823597335f384